src/main/resources/assets/js/app/issue/view/IssueListDialog.ts

```diff
@@ -288,8 +282,32 @@ export class IssueListDialog
         };
     }
 
-    private updateTabLabel(tabIndex: number, label: string, count: number) {
-        this.tabBar.getNavigationItem(tabIndex).setLabel(count > 0 ? (label + ' (' + count + ')') : label);
+    updateTabsLabels() {
+        this.updateAllTabLabel();
+        this.updatePublishRequestsTabLabel();
+        this.updateIssuesTabLabel();
+    }
+
+    private updateAllTabLabel() {
+        const total = this.allPanel.getActiveTotal();
+        const label = IssueListDialog.createTabLabel(i18n('field.all'), total);
+        this.allTab.setLabel(label);
+    }
+
+    private updatePublishRequestsTabLabel() {
+        const total = this.publishRequestsPanel.getActiveTotal();
+        const label = IssueListDialog.createTabLabel(i18n('field.publishRequests'), total);
+        this.publishRequestsTab.setLabel(label);
+    }
+
+    private updateIssuesTabLabel() {
+        const total = this.issuesPanel.getActiveTotal();
+        const label = IssueListDialog.createTabLabel(i18n('field.issues'), total);
+        this.issuesTab.setLabel(label);
+    }
+
+    private static createTabLabel(label: string, count: number) {
+        return count > 0 ? (label + ' (' + count + ')') : label;
     }
```

Comment:
`${label} (${count})` maybe?

Reply:
Yeah, using template string will be better. Fixed.